server/routes/reservationsRouter.js

```diff
@@ -10,6 +11,14 @@ reservationRouter.route('/')
       res.sendStatus(500);
     }
   })
+  .post(assignParking, async (req, res) => {
+    const inserted = await createReservation(req.body);
+    if (inserted) {
+      res.sendStatus(201);
+    } else {
+      res.sendStatus(500);
+    }
+  })
```

Comment:
create a reservation for a spot given passed in body, only consideration would be for future readability, only way to know what is being passed in through this function is to check the controller file, here a comment documenting what is expected to be passed along might be helpful.